CustomSkillForDataIngestion/QnAIntegrationCustomSkill/QnAIntegrationCustomSkill.cs

```diff
@@ -149,46 +149,63 @@ public static class UploadToQnAMaker
 
             await qnaClient.Knowledgebase.PublishAsync(kbId);
         }
-        private static async Task<string> GetKbID(ILogger log)
+        private static async Task<string> GetKbID(QnAMakerClient qnaClient, ExecutionContext context, ILogger log)
         {
-            // Create a BlobServiceClient object which will be used to create a container client
-            BlobServiceClient blobServiceClient = new BlobServiceClient(GetAppSetting("AzureWebJobsStorage"));
             string kbId = string.Empty;
-            BlobContainerClient containerClient = blobServiceClient.GetBlobContainerClient(Constants.kbContainerName);
-            BlobClient blobClient = containerClient.GetBlobClient(Constants.kbIdBlobName);
-            if (await blobClient.ExistsAsync())
+            var path = Path.Join(context.FunctionAppDirectory,"..\\qnamaker");
```

Comment:
D:\home\data\QnAMaker should be the path

Reply:
done